ruled_labels/specs_polkadot.yaml

```diff
@@ -0,0 +1,250 @@
+  - name: I0-consensus
+    description: Issue can lead to a consensus failure.
+    color: f6d9e9
+  - name: I1-panic
+    description: The node panics and exits without proper error handling.
+    color: 17b25d
+  - name: I2-security
+    description: "The node fails to follow expected, security-sensitive, behaviour."
+    color: D62209
+  - name: I3-bug
+    description: Fails to follow expected behavior.
+    color: FB3701
+  - name: I4-annoyance
+    description: "Code behaves within expectations, however this “expected behaviour” itself is at issue."
+    color: F76E02
+  - name: I5-tests
+    description: "Tests need fixing, improving or augmenting."
+    color: f3a603
+  - name: I6-documentation
+    description: "Documentation needs fixing, improving or augmenting."
+    color: fbca04
+  - name: I7-refactor
+    description: Code needs refactoring.
+    color: dbea04
+  - name: I8-footprint
+    description: "An enhancement to provide a smaller (system load, memory, network or disk) footprint."
+    color: 34C302
+  - name: I9-optimisation
+    description: An enhancement to provide better overall performance in terms of time-to-completion for a task.
+    color: 009B40
+  - name: J0-enhancement
+    description: An additional feature request.
+    color: 006b75
+  - name: J1-meta
+    description: A specific issue for grouping tasks or bugs of a specific category.
+    color: 0C7BAD
+  - name: J2-unconfirmed
+    description: "Issue might be valid, but it's not yet known."
+    color: "000000"
+  - name: J3-intended
+    description: Issue describes a behavior which turns out to work as intended. Closer should explain why.
+    color: b10d11
+  - name: J4-duplicate
+    description: Issue is a duplicate. Closer should comment with a link to the duplicate.
+    color: D62209
+  - name: J5-wont_fix
+    description: "Issue is in principle valid, but this project will not address it. Closer should explain why."
+    color: FB3701
+  - name: J6-invalid
+    description: Issue is invalid. Closer should comment why.
+    color: F76E02
+  - name: S0-design
+    description: PR/Issue is in the design stage
+    color: dd6e83
+  - name: S1-implement
+    description: PR/Issue is in the implementation stage
+    color: c9d7ff
+  - name: S2-test/monitor
+    description: PR/Issue is in the testing stage.
+    color: dbea04
+  - name: S3-deploy
+    description: PR/Issue is in the deployment stage
+    color: 34C302
+  - name: S4-blocked
+    description: "PR/Issue is blocked, see comments for further information."
+    color: D5C978
+  - name: S5-on_ice
+    description: "Work is currently on hold, see comments for further information."
+    color: "122381"
+  - name: T0-node
+    description: This PR/Issue is related to the topic “node”.
+    color: fbffe0
+  - name: T1-runtime
+    description: This PR/Issue is related to the topic “runtime”.
+    color: F5FCE6
+  - name: T2-API
+    description: This PR/Issue is related to APIs.
```

Comment:
Which API? RPC api? runtime api? rust crate api?

Reply:
we can extend the options here, right now it's very generic